test/Elastic.Apm.SqlClient.Tests/SqlServerFixture.cs

```diff
@@ -38,9 +39,21 @@ public async Task InitializeAsync()
 		public async Task DisposeAsync()
 		{
 			await _container.StopAsync();
-			_container.Dispose();
+			await _container.DisposeAsync();
 		}
 
-		public void Dispose() => _container?.Dispose();
+#if !NET5_0_OR_GREATER
+		private readonly ValueTask _completedValueTask = default;

```

Comment:
Is there a reason for making this a field?

Reply:
Probably just me overthinking :)

I got obsessed with `ValueTask.CompletedTask`, but that's only in .NET5+. So I wanted to have something similar and not new up a `ValueTask` directly. On the other hand, this is in the `DisposeAsync` method, which runs once, and the `_completedValueTask` field does not make any difference.

So I ended up removing it, and just do `return new ValueTask();`... it's much simpler in this scenario.

Reply:
Yes, easier to read. Thanks!